index.html

```diff
@@ -6,11 +6,204 @@
+                            <div class="mainheading">
+                                <p>
+                                    <span><em>ASTERISK ET OBELIX</em></span>
+                                    <span><em>CAKES!</em></span>
+                               </p>
+                           </div>
+                           <div class="thequote">
+                               <p><em>
+                                   <span>We Just Do Great Cakes!</span>
+                                   <span></span>
+                                   <span></span>The best cakes in town</span>
+                                   <span>delivered to your door</span>
+                               </em></p>
+                           </div>
+                       </header>
+
+
+                    </section>
+                </section>  
+                
+                <header class="menu-header">
+                    <nav class="navigation-bar">
+                        <ul>
+                            <li class="navigation-item">
+                                    <a class="navigation-link" href="#">HOME</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">CAKES</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">ORDERING</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">LESSONS</a>
+                            </li>
+                            <li class="navigation-item">
+                                <a class="navigation-link" href="#">ABOUT US</a>
+                          </li>
+                        </ul>
+                    </nav>
+                </header>                    
+        </section>
+
+        <section class="content-box">
+                <p>
+                    <span>We make all types of same-day cakes</span>
+                    <span>from Cupcakes, Celebration Cakes,</span>
+                    <span>Birthday Cakes, Chocolate cakes, Novelty cakes,</span>
+                    <span>Bespoke cakes and even same-day Wedding Cakes.</span>
+                    <span></span>
+                </p>
+
+                <p class="mobile-prompt">
+                     <!-- &#8595; is the symbol for a down-pointing arrow-->
+                    &#8595;Hover over the images below for more information&#8595;
+                </p>
+                
+                <p class="tablet-prompt">
+                    Hover over the image to the left and the images below for more information
+                </p>
+        </section>
+
+        <!-- NOTE: In order for this to work properly, the hierarchy and placement of these classes matters!! -->
+        <section class="cake-box">
```

Comment:
This section and the 4 cake images should be your main content and you can actually achieve a clean grid by having this structure:
```
<main class="main">
	<section class="main-image">
		<img alt="image1" class="images" src="image5.jpg"/>
	</section>
	<section class="main-text">
		Welcome and other texts
	</section>
	<section class="small-image image1">
		<img alt="image1" class="images" src="image1.jpg"/>
	</section>
	<section class="small-image image2">
		<img alt="image2" class="images" src="image2.jpg"/>
	</section>
	<section class="small-image image3">
		<img alt="image3" class="images" src="image3.jpg"/>
	</section>
	<section class="small-image image4">
		<img alt="image4" class="images" src="image4.jpg"/>
	</section>
</main>
```
You can apply the main class as a grid which contains 2 columns and 4 rows in mobile or 4 columns and 2 rows in desktop